mate-frontend.md

```diff
@@ -51,6 +51,34 @@ export const ChatDeprecated: FC<Props> // ...
 export const Chat: FC<Props> // ...
 ```
 
+#### 1.3 Create new components for A/B tests
```

Comment:
<img width="1165" alt="image" src="https://github.com/mate-academy/style-guides/assets/86183007/56d50959-cd40-484f-9549-f1b65303b443">

let's add it to the table of contents